src/SidePanel.tsx

```diff
@@ -16,7 +22,10 @@ export function SidePanel({
         {...props}
     >
         {
-            ["green", "blue", "red"].map(source => <li>{source}</li>)
+            solutions.map(x => <li className={cx(css({
```

Comment:
map passes counter as a second argument to callback: ••• .map((x, counter) => •••

Reply:
used map counter.